content/blog/2023/02/camel-k-roadmap-2023/index.md

```diff
@@ -0,0 +1,92 @@
+
+## Enhance build and make it more enterprise
+
+This is the area where we likely will introduce some important change. Let's start from one point discussed in the release process. As we need to decouple from the runtime, we'll need also to remove any dependency from the tooling required to build the application (which version change based on the runtime): more details are exposed in [#3831](https://github.com/apache/camel-k/issues/3831). This development may bring us to a new situation where we can enable a building strategy which by default will schedule a Job/Pod to run the build.
+
+Speaking about building strategy, something clear we have identified is the impossibility to support all the build strategy we have in place. We'll be deprecating Buildah, Kaniko and Spectrum favouring [JIB](https://github.com/apache/camel-k/issues/1656) which looks to be very solid and with a good community behind its back.
+
+Something else we need to rework is the way we generate the Camel project (the Maven project built from the Integration source). We have some custom code right now to do all the heavy lift but we could (should!?) use something similar done by `camel` CLI (Camel JBang). If we delegate this task to an external tool we'd benefit from 3 aspect:
+
+1. The Camel developer user experience is going to be the same independently on the final deployment choosen
+2. Local development will be easier and straightforward
+3. Something less to maintain... (just a joke, I promise we'll help Camel JBang instead)
+
+Having a generic project, may open some new interesting scenario. For example, at this stage we may be in a better position to support any Camel application, not only Camel Quarkus. And once we're there, the operator could even "intercept" any Camel application deployed by other tools (pipelines, manually, ...) and extend its operations (such as monitoring).
+
+In the same manner, the possibility to leverage the export feature provided by Camel JBang can lead us to a scenario where we can integrate more easily an external pipeline (ie, Tekton) and even think about some opinionated GitOps strategy to adopt.
+
+All these ideas should be furtherly analyzed and refined as they may clash against some assumptions we have already in place (for instance, reusing a container image kit). However, the main concept is that the way we build the application is a strategic piece and we'll be working to make it flexible and more stable in Camel K 2.0.
+
+## Multitenancy
+
+The new multitenancy model development has still some aspect to polish. One of them is the ability to have more than one operator running in the same namespace. We may also think if it makes sense to remove the global operator once we are confident with the new model.
```

Comment:
> remove the global operator

I think it should be `remove the namespace local operator`

Reply:
No I was really meaning the "global" one (it it makes sense, of course).

Reply:
No, the new model is keen to remove the namespace local operator so that all operators in future are "global" in terms of being able to reconcile resources on the whole cluster no matter of the namespace

Reply:
> all operators in future are "global"

I think we already discussed in the past the concept of "global" :) I'll remove this section so there is no misunderstanding.